darwin/Info.plist

```diff
@@ -22,5 +22,7 @@
 	<true/>
 	<key>LSBackgroundOnly</key>
 	<string>0</string>
+	<key>NSHighResolutionCapable</key>
+	<string>True</string>
```

Comment:
is there a reason we need this in addition to the mnemosyne.spec file changes below?

Reply:
The `macos` build process in the Makefile replaces the Info.plist, so I had to ensure the changes I made above weren't being overwritten. 
```
# from the macos build section in the Makefile....

# Custom Info.plist - may need to be updated periodically
	cp darwin/Info.plist dist/Mnemosyne.app/Contents

```

Reply:
ok, can you delete the custom Info.plist (and the corresponding line from the makefile), and add the NSSupportsAutomaticGraphicsSwitching  key via the mnemosyne.spec file per https://bugreports.qt.io/browse/QTBUG-29897? That way we can eliminate this duplication. 

The only reason this existed is because I didn't know about this better way of doing it via the spec file.